checks/collector.py

```diff
@@ -380,12 +380,15 @@ def run(self, checksd=None, start_event=True, configs_reloaded=False):
             if res:
                 metrics.extend(res)
 
+        # Use `info` log level for some messages on the first run only, then `debug`
+        log_method = log.info if self._is_first_run() else log.debug
```

Comment:
We should probably have a method `log_at_first_run`, unless this is the only thing doing this ?